Source/JavaScriptCore/runtime/ProxyObject.cpp

```diff
@@ -436,7 +436,7 @@ bool ProxyObject::performPut(JSGlobalObject* globalObject, JSValue putValue, JSV
     RETURN_IF_EXCEPTION(scope, false);
     if (!trapResultAsBool) {
         if (shouldThrow)
-            throwTypeError(globalObject, scope, makeString("Proxy object's 'set' trap returned falsy value for property '", String(propertyName.uid()), "'"));
+            throwTypeError(globalObject, scope, makeString("Proxy object's 'set' trap returned falsy value for property '"_s, String(propertyName.uid()), '\''));
```

Comment:
Should be able to pass `propertyName.uid()` without `String` to avoid reference count churn.